src/dataStrategies.js

```diff
@@ -58,8 +58,11 @@ export const results = ({
     )
     scrollId = result.context.scrollId
     page++
+    // We return _source flattened onto the root result items because we're mostly
+    // interested in the _source properties but may occasionally want other props like _id.
+    // This will be removed with #28 when a contexture-elasticsearch upgrade is complete
     return _.map(
-      F.getOrReturn('_source'),
+      r => _.isObject(r._source) ? ({ ..._.omit(['_source'], r), ..._.getOr({}, '_source', r) }): r._source ? r._source : r,
```

Comment:
With some fp fu, I think this can be reduced to:

```js
F.when('_source', r => ({ ..._.omit('_source', r), ...r._source }))
```

since if there is a _source, it'll always be an object

Reply:
not in the test -- should I update that?